pkg/services/ngalert/state/state.go

```diff
@@ -12,14 +12,15 @@ type State struct {
 	AlertRuleUID       string
 	OrgID              int64
 	CacheId            string
-	Labels             data.Labels
 	State              eval.State
 	Results            []Evaluation
 	StartsAt           time.Time
 	EndsAt             time.Time
 	LastEvaluationTime time.Time
 	EvaluationDuration time.Duration
+	LastSent           time.Time
```

Comment:
```suggestion
	LastSentAt           time.Time
```
nit: so that it matches the others